test/test_suite_test.go

```diff
@@ -264,9 +264,9 @@ var _ = BeforeAll(func() {
 				}
 			}
 		}
-		kubectl := helpers.CreateKubectl(helpers.K8s1VMName(), logger)
+		var kubectl *helpers.Kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
```

Comment:
I'm pretty sure `kubectl :=` is the same as `var kubectl *helpers.Kubectl`. Was the change just to make something more obvious?